### PR TITLE
[Fugu Status article] Remove stale list of APIs, replace with links to tracker

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Remove stale list of APIs, replace with links to tracker.

### CODE REVIEW

File: src/site/content/en/blog/fugu-status/index.md [hunk 1]
```diff
@@ -21,42 +16,16 @@ security, privacy, trust, and other core tenets of the web.
...
-          </ul>
-          <em>Updated April 14, 2020</em>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-</div>
+<a style="text-decoration: none;" class="w-button w-button--primary" href="https://fugu-tracker.web.app/#:~:text=Oct%2019%2C%202021)-,shipped%20%23,-Web%20Bluetooth%20API">
+  All already shipped APIs
+</a>
```
Comment (petele) [inline src/site/content/en/blog/fugu-status/index.md:28]: Should this be a button or a link?

Can you also ping Sam and see if he can fix the link so it works properly? :)

File: src/site/content/en/blog/fugu-status/index.md [hunk 2]
```diff
@@ -21,42 +16,16 @@ security, privacy, trust, and other core tenets of the web.
 You can see the full list of new and potential capabilities at
 the [Fugu API Tracker](https://goo.gle/fugu-api-tracker).
 
-## Capabilities available behind a flag {: #flag }
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/uIIvM9xocYkjmBfHSrJE.svg", alt="A fugu fish, the logo of Project Fugu.", width="150", height="150" %}
```
Comment (petele) [inline src/site/content/en/blog/fugu-status/index.md:19]: Can we put the image inline with the text so it's not floating in the middle of the page?

File: src/site/content/en/blog/fugu-status/index.md [hunk 3]
```diff
@@ -2,17 +2,12 @@
...
   - blog
   - capabilities
 ---
 
-{% Aside %}
-This is a living document and will be updated as features move between
-states.
-{% endAside %}
-
 The [capabilities project][capabilities-project] is a cross-company effort with the objective of
```
Comment (petele) [inline src/site/content/en/blog/fugu-status/index.md:11]: This section is almost identical to the subhead, maybe we should go into a bit more detail? Or list some of the apps that have used these new APIs and why? Something along the lines of This work allowed Adobe to bring PS to the web, Excalidraw to deprecate their Electron app, and some other really cool example.

File: src/site/content/en/blog/fugu-status/index.md (hunk 1 above)
Comment (tomayac) [reply src/site/content/en/blog/fugu-status/index.md:28]: These are links that shall look like buttons. In my tests they all worked, but the proper implementation depends on @Snugug merging https://github.com/GoogleChromeLabs/fugu-tracker/pull/26.

File: src/site/content/en/blog/fugu-status/index.md (hunk 2 above)
Comment (tomayac) [inline src/site/content/en/blog/fugu-status/index.md:19]:
```suggestion
{% Img style="float: left;" src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/uIIvM9xocYkjmBfHSrJE.svg", alt="A fugu fish, the logo of Project Fugu.", width="150", height="150" %}
```

Comment (tomayac) [reply src/site/content/en/blog/fugu-status/index.md]: Done.

File: src/site/content/en/blog/fugu-status/index.md [hunk 4]
```diff
@@ -2,61 +2,34 @@
...
 tags:
   - blog
   - capabilities
 ---
 
-{% Aside %}
-This is a living document and will be updated as features move between
-states.
-{% endAside %}
+{% Img style="float: left", src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/uIIvM9xocYkjmBfHSrJE.svg", alt="A fugu fish, the logo of Project Fugu.", width="150", height="150" %}
```
Comment (tomayac) [inline src/site/content/en/blog/fugu-status/index.md:11]:
```suggestion
{% Img style="float: right", src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/uIIvM9xocYkjmBfHSrJE.svg", alt="A fugu fish, the logo of Project Fugu.", width="150", height="150" %}
```

File: src/site/content/en/blog/fugu-status/index.md [hunk 5]
```diff
@@ -2,61 +2,34 @@
...
 tags:
   - blog
   - capabilities
 ---
 
-{% Aside %}
-This is a living document and will be updated as features move between
-states.
-{% endAside %}
+{% Img style="float: left" src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/uIIvM9xocYkjmBfHSrJE.svg", alt="A fugu fish, the logo of Project Fugu.", width="150", height="150" %}
```
Comment (tomayac) [inline src/site/content/en/blog/fugu-status/index.md]:
```suggestion
{% Img style="float: left; margin-inline: 10px;" src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/uIIvM9xocYkjmBfHSrJE.svg", alt="A fugu fish, the logo of Project Fugu.", width="150", height="150" %}
```

File: src/site/content/en/blog/fugu-status/index.md [hunk 6]
```diff
@@ -2,61 +2,34 @@
...
 tags:
   - blog
   - capabilities
 ---
 
-{% Aside %}
-This is a living document and will be updated as features move between
-states.
-{% endAside %}
+{% Img style="float: right", src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/uIIvM9xocYkjmBfHSrJE.svg", alt="A fugu fish, the logo of Project Fugu.", width="150", height="150" %}
```
Comment (tomayac) [inline src/site/content/en/blog/fugu-status/index.md:11]:
```suggestion
{% Img style="float: right; margin-inline: 10px;" src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/uIIvM9xocYkjmBfHSrJE.svg", alt="A fugu fish, the logo of Project Fugu.", width="150", height="150" %}
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**